command-line/test/Command/Address/InspectSpec.hs

```diff
@@ -39,6 +39,14 @@ spec = describeCmd [ "address", "inspect" ] $ do
     specInspectAddress ["Shelley", "by pointer"]
         "addr1gw2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzer5ph3wczvf2x4v58t"
 
+    -- reward account: keyhash28
+    specInspectAddress ["Shelley", "by value"]
+        "addr1upshvetj09hxjcm9v9jxgunjv4ehxmr0d3hkcmmvdakx7mq726p32"
+
+    -- reward account: scripthash32
+    specInspectAddress ["Shelley", "by value"]
+        "addr17pshvetj09hxjcm9v9jxgunjv4ehxmr0d3hkcmmvdakx7mrgdp5xsnxe5e6"
```

Comment:
bech32 encoding is only a presentation layer so this works just fine. Yet in principle, stake addresses should use the prefix `stake` to distinguish them from standard addresses. We have started to work on a list of agreed prefixes to facilitate interop between the various tools we have, so have a quick look at: https://github.com/cardano-foundation/CIPs/tree/master/CIP5